.github/workflows/push-tests.yml

```diff
@@ -222,16 +222,10 @@ jobs:
             **/Dockerfile
             requirements*.txt
             **/requirements*.txt
-      - name: Check if containers need to be rebuilt
-        if: steps.changed-files.outputs.any_changed == 'true'
-        run: |
-          echo "DOCKER_REPO=${DOCKER_REPO}" > .env.ecr
-          make local-rebuild-backend
-      - name: Run tests in docker-compose
+      - name: Run unit tests
```

Comment:
Is adding an integration test step part of the scope of this PR, or should it be added in a different PR?

Reply:
I would say different PR. This should be merged as soon as possible to avoid further PRs to break tests.